ui/admin/app/templates/scopes/scope/targets/target/index.hbs

```diff
@@ -30,9 +30,74 @@
           @cancel={{route-action 'cancel' @model}}
         />
       </bc.Body>
-      {{#if (and @model.isSSH (feature-flag 'ssh-session-recording'))}}
-        <bc.Sidebar>
-          <div class='target-sidebar'>
+
+      <bc.Sidebar>
+        <div class='target-sidebar-aliases'>
+          <h3>{{t 'resources.alias.title_plural'}}</h3>
+          <p class='description'>
+            {{t 'resources.alias.form.sidebar.help'}}
+          </p>
+          {{#if @model.aliases}}
+
+            {{#each this.aliases as |alias|}}
+              <LinkListPanel as |P|>
+                <P.Item
+                  @icon='at-sign'
+                  @color='boundary'
+                  @text={{alias.value}}
+                  {{! @route=' ' }}
+                  @model={{alias.id}}
+                />
+              </LinkListPanel>
+            {{/each}}
+
+            {{#if this.showFlyoutBtn}}
+              <Hds::Button
+                data-test-view-more
+                @text={{t 'resources.target.actions.view-more-aliases'}}
+                @color='secondary'
+                {{on 'click' this.targets.toggleFlyout}}
+              />
+            {{/if}}
+            {{#if this.showFlyout}}
+              <Hds::Flyout
+                data-test-flyout
+                @onClose={{this.toggleFlyout}}
+                as |M|
+              >
+                <M.Header>
+                  {{t
+                    'resources.target.titles.aliases-flyout'
+                    targetDisplayName=@model.displayName
+                  }}
+                </M.Header>
+
+                <M.Body>
+                  {{#each this.remainingAliases as |alias|}}
+                    <LinkListPanel as |P|>
+                      <P.Item
+                        @icon='at-sign'
+                        @color='boundary'
+                        @text={{alias.value}}
+                        {{! @route='' }}
+                        @model={{alias.id}}
+                      />
+                    </LinkListPanel>
+                  {{/each}}
+                </M.Body>
+              </Hds::Flyout>
+            {{/if}}
+          {{else}}
+            <Hds::Button
+              @icon='plus'
+              @text={{t 'resources.target.actions.add-an-alias'}}
+              @color='secondary'
+              {{! @route='' }}
+            />
+          {{/if}}
```

Comment:
Can a target can have more than one alias? If that's correct doesn't this else check prevent you from adding more if an alias is added?

Reply:
A target can have multiple aliases only at creation time in the target model. To add an alias within the alias domain/model, we use this button, but in the alias model we are allowed to add only one alias at a time.

Reply:
yes, it does. the target route will have this update